packages/lumx-react/src/components/select/Select.tsx

```diff
@@ -238,233 +49,128 @@ const stopPropagation = (evt: Event) => evt.stopPropagation();
-                                </InputLabel>
-                            </div>
-                        )}
-
-                        <div
-                            ref={anchorRef as RefObject<HTMLDivElement>}
-                            id={targetUuid}
-                            className={`${CLASSNAME}__wrapper`}
-                            onClick={onInputClick}
-                            onKeyDown={handleKeyboardNav}
-                            tabIndex={0}
-                        >
-                            <div className={`${CLASSNAME}__chips`}>
-                                {!isEmpty && isMultiple && (
-                                    <ChipGroup>
-                                        {value.map((val: string, index: number) =>
-                                            selectedChipRender!(val, index, onClear, isDisabled),
-                                        )}
-                                    </ChipGroup>
-                                )}
-                            </div>
-
-                            {isEmpty && placeholder && (
-                                <div
-                                    className={classNames([
-                                        `${CLASSNAME}__input-native`,
-                                        `${CLASSNAME}__input-native--placeholder`,
-                                    ])}
-                                >
-                                    <span>{placeholder}</span>
-                                </div>
-                            )}
-
-                            {!isEmpty && !isMultiple && (
-                                <div className={`${CLASSNAME}__input-native`}>
-                                    <span>{selectedValueRender!(value[0])}</span>
-                                </div>
-                            )}
-
-                            {(isValid || hasError) && (
-                                <div className={`${CLASSNAME}__input-validity`}>
-                                    <Icon icon={isValid ? mdiCheckCircle : mdiAlertCircle} size={Size.xxs} />
-                                </div>
-                            )}
-
-                            {hasInputClear && (
-                                <IconButton
-                                    className={`${CLASSNAME}__input-clear`}
-                                    icon={mdiCloseCircle}
-                                    emphasis={Emphasis.low}
-                                    size={Size.s}
-                                    theme={theme}
-                                    onClick={onClear}
-                                    onKeyDown={stopPropagation}
-                                />
-                            )}
-
-                            <div className={`${CLASSNAME}__input-indicator`}>
-                                <Icon icon={mdiMenuDown} size={Size.s} />
-                            </div>
+}): React.ReactElement => {
```

Comment:
The return type here is not required since it is provided in the `React.FC` type signature